src/index.html

```diff
@@ -8,5 +8,38 @@
   </head>
   <body>
     <h1>Calendar</h1>
+    <div class="calendar calendar--month-length--28">
+      <div class="calendar__cell calendar__cell--start-day--sun"></div>
```

Comment:
both of your modifiers should be applied to parent element
```suggestion
    <div class="calendar calendar--month-length--28 calendar--start-day--sun">
      <div class="calendar__cell"></div>
```

Reply:
both modifiers applied to parent element, but got lint errors, can't understand how change modifiers
![image](https://user-images.githubusercontent.com/117356906/214780117-d9b88b6c-4229-4e7b-a9ad-eb333907c1fb.png)